File: chat-helper/src/main/java/com/mpls/v2/helper/model/Message.java
package com.mpls.v2.helper.model;

/**
 * Created by danul on 16.10.2017.
 */
public class Message {

    private Boolean isCallCenter;
    private String text;
    private String name;
    private Integer chatRoomId;
    private Boolean create;

    public Boolean getCreate() {
        return create;
    }

    public void setCreate(Boolean create) {
        this.create = create;
    }

    public Boolean getCallCenter() {
        return isCallCenter;
    }

    public Message setCallCenter(Boolean callCenter) {
        isCallCenter = callCenter;
        return this;
    }

    public Integer getChatRoomId() {
        return chatRoomId;
    }

    public Message setChatRoomId(Integer chatRoomId) {
        this.chatRoomId = chatRoomId;
        return this;
    }

    public String getText() {
        return text;
    }

    public Message setText(String text) {
        this.text = text;
        return this;
    }

    public String getName() {
        return name;
    }

    public Message setName(String name) {
        this.name = name;
        return this;
    }

    @Override
    public String toString() {
        return "Message{" +
                "text='" + text + '\'' +
                '}';
    }
}

File: public/front/src/environments/model/message.ts
/**
 * Created by danul on 16.10.2017.
 */
export class Message {
  public text: string;
  public name: string;
  public CallCenter: boolean;
  public create: boolean;
  public chatRoomId: number;

  constructor(text?: string, name?: string, CallCenter?: boolean, create?: boolean, chatRoomId?: number) {
    this.text = text;
    this.name = name;
    this.CallCenter = CallCenter;
    this.chatRoomId = chatRoomId;
    this.create = create;
  }
}
